plugins/lookup/lookup.py

```diff
@@ -61,6 +61,10 @@
             description:
                 - The api_filter to use.
             required: False
+        api_version:
+            description:
+                - The Nautobot Rest API version to use.
+            required: False
```

Comment:
Need to add in `version_added: "3.0.0`
```suggestion
        api_version:
            description:
                - The Nautobot Rest API version to use.
            required: False
            version_added: "4.0.1"
```